src/main/java/org/spongepowered/api/util/command/args/GenericArguments.java

```diff
@@ -1126,8 +1119,8 @@ protected CatalogedTypeCommandElement(Text key, Game game, Class<T> catalogType)
         protected Iterable<String> getChoices(CommandSource source) {
             return this.game.getRegistry().getAllOf(this.catalogType).stream()
                 .map(input -> {
-                return input == null ? null : input.getId(); // TODO: ids or names?
-                })
+                        return input == null ? null : input.getId(); // TODO: ids or names?
```

Comment:
Updating checkstyle to a newer version now that we use Java 8 will probably fix this, will look into doing that soon. Up to now we were stuck on an older checkstyle version because newer versions require Java 7+